### DISCUSSION TITLE
how to implement global pooling for edge_attr

### DISCUSSION THREAD

**Discussion by Powerd0g** [pyg-team/pytorch_geometric#4634] [Q&A]
example:
DataBatch(x=[9, 11], edge_index=[2, 14], edge_attr=[14, 4], batch=[9])
in this situation, there are 9 nodes and 14 edges for 2 batch
i use the `global_add_pool(x=x, batch=batch)` to global pooling the nodes,
So i want to know how to global pooling the edges.

or could you tell me how to get the batch of the edges

**Comment by rusty1s** [comment]
You can use the following code snippet:
```python
edge_batch = batch[edge_index[0]]
out = global_add_pool(edge_attr, edge_batch)
```
